### PR TITLE
Required for Live Analytics in order to make the liveStats->collect()…

### PR DESCRIPTION
**Pull request by jessp01**
… request.

### CODE REVIEW

File: configurations/html5.template.php [hunk 1]
```diff
@@ -28,6 +28,10 @@
 $wgKalturaCDNUrl = wgGetUrl('cdn_host');
 // Default Stats URL
 $wgKalturaStatsServiceUrl = wgGetUrl('stats_host');
+// Default Live Stats URL
+$wgKalturaLiveStatsServiceUrl = wgGetUrl('live_stats_host');
+// Default Kaltura Analytics URL
+$wgKalturaAnalyticsServiceUrl = wgGetUrl('analytics_host');
 
 // SSL host names
 if( $wgHTTPProtocol == 'https' ){
```
Comment (zoharbabin) [inline configurations/html5.template.php]: isn't that what's done 3 lines below?
https://github.com/kaltura/server/blob/42201ef841cd5c0ff98d4dc3d98c182576a19a29/configurations/html5.template.php#L37

Comment (jessp01) [inline configurations/html5.template.php]: Indeed it is, line 40:
$wgKalturaStatsServiceUrl = wgGetUrl('stats_host_https');

Comment (yossipapi) [inline configurations/html5.template.php]: see the difference:
$wgKalturaLiveStatsServiceUrl = wgGetUrl('live_stats_host_https');
is not 
$wgKalturaStatsServiceUrl = wgGetUrl('stats_host_https');

Comment (jessp01) [inline configurations/html5.template.php]: Added:
$wgKalturaLiveStatsServiceUrl = wgGetUrl('live_stats_host_https');
See line 41.

Thanks,

**Comment by yossipapi** [comment]
Created https://github.com/kaltura/server/pull/4984 to merge to latest working branch.

**Pull request closed without merging**